blog.html

```diff
@@ -22,6 +22,119 @@
+
+                </ul>
+                <h3>Summary</h3>
+                <p>Ready to get started with Apache Kafka 3.7.0? Check out all the details in the <a href="https://archive.apache.org/dist/kafka/3.7.0/RELEASE_NOTES.html">release notes</a> and <a href="https://kafka.apache.org/downloads">download</a> Apache Kafka 3.7.0.</p>
+
+                <p>This was a community effort, so thank you to everyone who contributed to this release, including all our users and our 157 contributors: <br>
+                    A. Sophie Blee-Goldman, Abhijeet Kumar, Akhilesh Chaganti, Alieh, Alieh Saeedi, Almog Gavra, Alok Thatikunta, Alyssa Huang, Aman Singh, Andras Katona, Andrew Schofield, Anna Sophie Blee-Goldman, Anton Agestam, Apoorv Mittal, Arnout Engelen, Arpit Goyal, Artem Livshits, Ashwin Pankaj, ashwinpankaj, atu-sharm, bachmanity1, Bob Barrett, Bruno Cadonna, Calvin Liu, Cerchie, chern, Chris Egerton, Christo Lolov, Colin P. McCabe, Colin Patrick McCabe, Colt McNealy, Crispin Bernier, David Arthur, David Jacot, David Mao, dengziming, Deqi Hu, Dimitar Dimitrov, Divij Vaidya, Dongnuo Lyu, Eaugene Thomas, Eduwer Camacaro, Eike Thaden, Federico Valeri, Federico Valeri fedevaleri@gmail.com, Ziming Deng dengziming1993@gmail.com, Mickael Maison mimaison@apache.org., Florin Akermann, Gantigmaa Selenge, Gaurav Narula, gongzhongqiang, Greg Harris, Guozhang Wang, Gyeongwon, Do, Hailey Ni, Hanyu Zheng, Hao Li, Hector Geraldino, hudeqi, Ian McDonald, Iblis Lin, Igor Soarez, iit2009060, Ismael Juma, Jakub Scholz, James Cheng, Jason Gustafson, Jay Wang, Jeff Kim, Jim Galasyn, John Roesler, Jorge Esteban Quilcate Otoya, Josep Prat, José Armando García Sancio, Jotaniya Jeel, Jouni Tenhunen, Jun Rao, Justine Olshan, Kamal Chandraprakash, Kirk True, kpatelatwork, kumarpritam863, Laglangyue, laglangyue, Levani Kokhreidze, Lianet Magrans, Liu Zeyu, Lucas Brutschy, Lucia Cerchie, Luke Chen, maniekes, Manikumar Reddy, mannoopj, Maros Orsak, Matthew de Detrich, Matthias J. Sax, Max Riedel, Mayank Shekhar Narula, Mehari Beyene, Michael Westerby, Mickael Maison, Nick Telford, Nikhil Ramakrishnan, Nikolay, Okada Haruki, olalamichelle, Omnia G H Ibrahim, Omnia G.H Ibrahim, Omnia Ibrahim, Owen Leung, Paolo Patierno, Philip Nee, Phuc-Hong-Tran, phuchong.tran, Proven Provenzano, Purshotam Chauhan, Qichao Chu, Qichao Chu (@ex172000), Matthias J. Sax, Rajini Sivaram, Renaldo Baur Filho, Reviewers: David Jacot, Reviewers: Divij Vaidya, Ritika Reddy, Robert Wagner, Rohan, Ron Dagostino, Roon, runom, Ruslan Krivoshein, rykovsi, Sagar Rao, Said Boudjelda, Satish Duggana, shuoer86, Stanislav, Stanislav Kozlovski, Taher Ghaleb, Tang Yunzi, TapDang, Taras Ledkov, tkuramoto33, Tyler Bertrand, vamossagar12, Vedarth Sharma, Viktor Somogyi-Vass, Vincent Jiang, Walker Carlson, Wuzhengyu97, Xavier Léauté, Xiaobing Fang, yangy0000, Ritika Reddy, Yanming Zhou, Yash Mayya, yuyli, zhaohaidao, Zihao Lin, Ziming Deng, Ziming Deng dengziming1993@gmail.com, Ziming Deng dengziming1993@gmail.com.
```

Comment:
The script that generates that list is a bit naive so you usually want to do some manual cleanups (and adjust the count above).
- Ziming Deng appears at least 5 times `dengziming`, `Ziming Deng dengziming1993@gmail.co`, `Ziming Deng, Ziming Deng dengziming1993@gmail.com`, `Ziming Deng dengziming1993@gmail.com`
- `Federico Valeri fedevaleri@gmail.com, Ziming Deng dengziming1993@gmail.com, Mickael Maison mimaison@apache.org.` can be removed as each appear also in the list
- `A. Sophie Blee-Goldman`, `Anna Sophie Blee-Goldman`
- `Colin P. McCabe`, `Colin Patrick McCabe`
- `Reviewers: David Jacot, Reviewers: Divij Vaidya` can be removed, they also appear earlier in the list
- `Omnia G H Ibrahim`, `Omnia G.H Ibrahim`, `Omnia Ibrahim`

You don't want to spend much time to double check all names but a quick clean up is nice

Reply:
Makes sense. Thanks for pointing it out. I actually didn't spot-check it